build/xtask/src/config.rs

```diff
@@ -23,6 +23,7 @@ struct RawConfig {
     target: String,
     board: String,
     chip: String,
+    fmc: Option<FmcConfig>,
```

Comment:
So, "fmc" is a term specific to certain ST microcontrollers... can we think of a more general name for this in the shared config?

Reply:
switched to `mmio`